x-pack/solutions/observability/plugins/apm/public/components/routing/templates/apm_main_template/entities_inventory_callout.tsx

```diff
@@ -49,6 +50,9 @@ export function EntitiesInventoryCallout({ onDismiss }: EntitiesInventoryCallout
         >
           <EuiButtonIcon
             data-test-subj="apmEntitiesInventoryCalloutDismiss"
+            aria-label={i18n.translate('xpack.apm.entitiesInventoryCallout.linkTooltip', {
+              defaultMessage: 'Hide this',
```

Comment:
NIT Maybe something like this to be more descriptive? WDYT? It is also OK to stay like it is
```suggestion
              defaultMessage: 'Hide the entities inventory callout',
```

Reply:
much better! thanks